### PR TITLE
[CI:DOCS] GHA: Simplify Cirrus-Cron check slightly

### PR DESCRIPTION
**Pull request by cevich**
Depends on: https://github.com/containers/podman/pull/16357 and https://github.com/containers/podman/pull/16414

Signed-off-by: Chris Evich <cevich@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: .github/workflows/check_cirrus_cron.yml [hunk 1]
```diff
@@ -62,24 +62,7 @@ jobs:
...
-                    echo "Cron build '$NAME' Failed: https://cirrus-ci.com/build/$BID"
-                done < "$NAME_ID_FILEPATH"
-
-                echo ""
-                echo "# Source: ${{ github.workflow }} workflow on ${{ github.repository }}."
-                # Separate content from sendgrid.com automatic footer.
-                echo ""
-                echo ""
-                ) > ./artifacts/email_body.txt
+              run: './_podman/.github/actions/check_cirrus_cron/make_email_body'
```
Comment (rhatdan) [inline .github/workflows/check_cirrus_cron.yml:65]: Does this script exists in Buildah?

Comment (cevich) [reply .github/workflows/check_cirrus_cron.yml:65]: Nope, it's in podman.  It's also broken, so this PR should depend on https://github.com/containers/podman/pull/16414

**Comment by cevich** [comment]
This PR should be ready.

**Comment by cevich** [comment]
This is ready now.

**Pull request merged**